.github/dependabot.yaml

```
@@ -24,24 +24,16 @@ updates:
     directory: /
     schedule:
       interval: daily
+    ignore:
+      - dependency-name: "torch"
+      - dependency-name: "lightning"
+      - dependency-name: "pytorchcv"
+      - dependency-name: "timm"
+      - dependency-name: "openvino*"
+      - dependency-name: "nncf"
+      - dependency-name: "anomalib"
```

```suggestion
      - dependency-name: "anomalib"
      - dependency-name: "intel-extension-for-pytorch"
```
I think it's better to add IPEX also.